tests/functional_tests/functional_tests.py

```diff
@@ -1,11 +1,12 @@
 from __future__ import unicode_literals
+import base64
```

Comment:
I'd like to see a test that deserializes some encoded macaroons that have been produced
by
- the older version of pymacaroons
- the Go package
- libmacaroons

all with binary and json formats.
And also check that the macaroons pass verification after deserializing, and round trip successfully too.